contributor_docs/friendly_error_system.md

```diff
@@ -135,10 +135,10 @@ These functions are mainly responsible for catching errors and generating FES me
 
 For full reference, please see our [Dev Notes].
 
-[`_friendlyFileLoadError()`]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md#_friendlyfileloaderror
-[`_validateParameters()`]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md#validateparameters
-[`_fesErrorMontitor()`]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md#feserrormonitor
-[Dev Notes]: https://github.com/processing/p5.js/blob/main/contributor_docs/fes_reference_dev_notes.md
+[`_friendlyFileLoadError()`]: https://github.com/processing/p5.js/blob/daba49806a389e98cd63c62667928f61243c7fa1/contributor_docs/fes_contribution_guide.md#_friendlyerror
```

Comment:
Hi, thanks for catching this. I think you can keep the 'main' in the URL, so the fixed the link would be like the following:
https://github.com/processing/p5.js/blob/main/contributor_docs/fes_contribution_guide.md#_friendlyerror

cc @limzykenneth